src/main/drivers/adc.h

```diff
@@ -18,12 +18,40 @@
 #pragma once
 
 typedef enum {
-    ADC_BATTERY = 0,
-    ADC_RSSI = 1,
+    ADC_CHANNEL1_BIT = 0,
+    ADC_CHANNEL2_BIT = 1,
+    ADC_CHANNEL3_BIT = 2,
+    ADC_CHANNEL4_BIT = 3,
+} adcChannelBit_e;
+
+typedef enum {
+    ADC_CHANNEL1_ENABLE = (1 << ADC_CHANNEL1_BIT),
+    ADC_CHANNEL2_ENABLE = (1 << ADC_CHANNEL2_BIT),
+    ADC_CHANNEL3_ENABLE = (1 << ADC_CHANNEL3_BIT),
+    ADC_CHANNEL4_ENABLE = (1 << ADC_CHANNEL4_BIT),
+} adcChannelEnableMask_e;
```

Comment:
Is this really necessary? One of dcChannelBit_e / adcChannelEnableMask_e should be enough ...